dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -1561,6 +1561,11 @@ NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W=CNTR0340W: The {2} Java Naming an
 NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W.explanation=The local binding name can only contain one namespace and it must start with ejblocal:. Namespaces are denoted by the colon (:) character. The entire string before and including the colon (:) character is considered a namespace.
 NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W.useraction=Correct the ibm-ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the namespace that is not valid.
 
+#-------------------------------------------------------------------------------
+UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W=CNTR0341W: Unable to bind the {0} interface of the {1} bean in the {2} module of the {3} application to the {4} name location. The ORB service is unavailable.
+UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.explanation=Could not create the remote enterprise bean binding because the ORB service is unavailable
```

Comment:
If @dmuelle agrees, I'd recommend removing "service" and switch to "not available" and perhaps rework to use cannot like

The remote enterprise bean binding cannot be created because the ORB is not available.

Reply:
So, perhaps:
```
The remote enterprise bean binding cannot be created because the CORBA name server is not available.
```

Reply:
The latter LGTM- in general, code entities should have a noun descriptor after them to help with translations. So we have CORBA _name server_ here which is good